studio/components/to-be-cleaned/Docs/Snippets.js

```diff
@@ -543,7 +543,7 @@ curl -X POST '${endpoint}/auth/v1/otp' \\
     js: {
       language: 'js',
       code: `
-let { user, error } = await supabase.auth.signIn({
+let { user, error } = await supabase.auth.signInWithOtp({
```

Comment:
@kangmingtay is `signInWithOtp` the correct method to replace this?

Reply:
yup